javascript-data-model/main.js

```diff
@@ -0,0 +1,24 @@
+/* exported orderHistory */
+
+var orderHistory = [
+  {
+    order1Header: [{ orderNo: '114-3941689-8772232' }, { datePlaced: 'August 4, 2020' }, { total: 34 }, { shipTo: 'JS Masher' }, { delivered: '08/08/2020' }],
```

Comment:
Rather than naming these properties individually for each order such as `order1Header`, you should use a general name that applies to all orders, such as `orderDetails`. Additionally, this is an array of objects that each have one property. This would be a perfect opportunity to remove the array, and just make this one object with all the necessary properties.